installer/opensuse/releases

```diff
@@ -0,0 +1,3 @@
+tumbleweed
+leap-42.2
+leap-42.3
```

Comment:
and we don't want just a "leap" release that grabs the latest version, right?

Reply:
no, these are independent releases.